Ajoute des articles d’aide

bboulesteix opened the pull request:
Cette _pull-request_ :

1. Enrichit le site [doc.data.gouv.fr](https://doc.data.gouv.fr) avec de nouveaux articles d’aide, comme annoncé dans #36 ;
2. Créé un dossier `articles` dans lequel sont rassemblées les collections ([voir la doc](https://jekyllrb.com/docs/collections/)) ;
3. Supprime les passages caducs, ceux qui parlaient de l’année 2015 au _futur_.

Votre relecture sur les articles ajoutés est la bienvenue — des coquilles doivent encore s’y cacher.👀 

![capture d ecran 2018-10-15 18 46 49_shrink](https://user-images.githubusercontent.com/3286488/46965564-7d998c00-d0ab-11e8-91ba-89cabb89edb3.png)

bboulesteix committed:
Ajoute des articles à la documentation

Ce commit enrichit la documentation de data.gouv.fr avec de
nouveaux articles, répartis dans de nouvelles catégories. Il
définit aussi un emplacement pour toutes les collections, dans
le but de rassembler tous les articles dans un même dossier.

bboulesteix committed:
Supprime les caducs ou datés

Ce commit retire les passages qui parlaient de 2015 au futur.

bboulesteix committed:
Supprime les notes de travail du rédacteur

Ce commit supprime les commentaires laissés par Bibi pour savoir
comment relier les articles d’aide entre eux une fois ces derniers
mis en ligne. Il s’agit de commentaires de travail qui ne doivent
pas apparaître en ligne.

noirbizarre commented on articles/_faq/producteur.md:
Il faudrait voir avec l'équipe geo si la suite est toujours d'actualité ou s'il faut aussi supprimer le paragraphe suivant.

bboulesteix replied:
Toute la partie _faq_ va progressivement être remplacée par les articles d’aide annoncés dans #36. Dans un premier temps, j’ai retiré les passages vraiment caducs, mais à terme c’est l’intégralité de l’article qui va disparaître.

noirbizarre commented on articles/_gestion-du-compte/fermer-un-compte.md:
```diff
@@ -0,0 +1,15 @@
+4. Cliquer sur la flèche située à côté du bouton **Éditer**, en haut à droite de votre écran, puis sur **Supprimer le profil** dans le menu déroulant qui apparaît alors ;
+5. Confirmez la suppression de votre compte en cliquant sur le bouton **Confirmer**, qui figure dans la boîte de dialogue rouge.
+
+La fermeture d’un compte est irréversible. Les informations associées à un compte supprimé sont définitivement effacées.
```
Préciser que le contenu publié ne le sera pas, qu'il sera simplement anonymisé.

bboulesteix replied:
Merci pour cette précision @noirbizarre.

Avant que je ne modifie le fichier en question, est-ce que la formulation suivante te semble juste ?

> Les données publiées par le compte qui fait l’objet de la fermeture ne seront pas supprimées, elles resteront en ligne, mais sous une forme anonyme, sans producteur désigné.

noirbizarre replied:
Cela comprend aussi les discussions et les réutilisations, donc la notion de producteur doit s'effacer un peu.
Contre-proposition:
> La suppression d'un compte est irréversible. Les contenus publiées par ce compte ne seront pas supprimées, ils resteront en ligne sous une forme anonymisée:
> - les jeux de données et les réutilisations n'auront plus de producteur ou auteur associé
> - l'utilisateur sera anonymisé dans les discussions, les anomalies et les signalements

Par contre j'ai volontairement remplacé "fermeture" par "suppression" pour être cohérent avec ce qui précède et surtout pour être en conformité avec la CNIL (la fermeture n'implique pas la suppression du compte)

bboulesteix replied:
Ça me va, merci pour ces précisions. 👍

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+* sous votre propre nom, à titre individuel ;
+* pour le compte d’une organisation, à titre collectif.
+
+L’étape 1 de la publication d’un jeu de données vous permet de choisir si les données doivent être publiées à votre nom ou sous la bannière de votre organisation.
```
Je pense qu'il faudrait encore plus insister sur ce point, avec un **Attention** quelque part pour vraiment faire en sorte de les orga publient en tant qu'orga et pas sous un compte perso unique

noirbizarre replied:
Peut-être rajouter un lien vers la doc des orgas plus bas qui a plus de contexte

bboulesteix replied:
Merci pour cette précision @noirbizarre.

> Je pense qu'il faudrait encore plus insister sur ce point, avec un Attention quelque part pour vraiment faire en sorte de les orga publient en tant qu'orga et pas sous un compte perso unique.

**Question** : est-ce à dire que beaucoup d’utilisateurs publient des données sous leur propre nom alors qu’ils pensaient publier pour le compte d’une organisation à laquelle ils appartiennent ?

> Peut-être rajouter un lien vers la doc des orgas plus bas qui a plus de contexte.

Une fois que tous les articles d’aide évoqués dans #36 seront en ligne, j’ai prévu de créer un maillage entre eux. Je vais ajouter, pour chaque article, une section **À lire ensuite** au bas de chaque article, pour relier entre eux les articles qui se complètent tout en se précisant.

romtal replied:
On a eu de nombreux cas de producteurs publiant sous leur propre nom malheureusement...c'est un cas assez courant...la distinction entre compte utilisateur et organisation productrice doit être clairement explicité...

bboulesteix replied:
D’accord, c’est bien ce qui me semblait @romtal, merci pour cette précision. Si beaucoup d’utilisateurs s’emmêlent les pinceaux entre leur compte personnel d’une part, et celui de leur organisation d’autre part, la documentation ne pourra hélas qu’y remédier à la marge. 🙁 

Je veux bien ajouter quelques précisions sur ce point dans l’article d’aide en question, mais peut-être s’agit-il là d’un problème d’interface plus profond ? Je ne sais pas.

noirbizarre replied:
On a déjà fait quelques changement ergonomique sur l'admin pour inciter à utiliser une orga, malgré cela,on en a encore qui **volontairement** publient avec un compte générique alors qu'il faut, au choix suivant les cas:
- volontairement sélectionner son compte utilisateur alors que c'est l'orga qui est pré-sélectionnée
- passer outre les différents rappels, encarts rouges... qui pavent le parcours de publication des utilisateurs sans orga

Je continue donc à penser que c'est volontaire sur les quelques cas qui restent. Un rappel ici du pourquoi/comment ne me parait pas superflu

bboulesteix replied:
Merci pour vos explications @noirbizarre et @romtal. J'ai ajouté une phrase qui va dans votre sens. ✅

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+* **Mauvais titre** : Horaires des cars
+* **Bon titre** : Horaires des autobus de la ville de Brive
+
+La présence d’un titre est obligatoire.
```
Préciser que le titre est le premier élement qui sert à la découvrabilité du jeu de données, notement en recherche

bboulesteix replied:
Bon point, tu as raison.

noirbizarre replied:
Le point est marqué comme corrigé, mais il n'y a pas les détails évoqués

bboulesteix replied:
Hum, pourtant le [commit en question](https://github.com/etalab/doc.data.gouv.fr/commit/e2d4cbba8761c176117f39ca2b849300c4e78d6a#diff-671045a24e5b5daaed489b42881e55dc) semble bien passé. Peut-être que les aperçus de GitHub ne sont pas frais. 🤔

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+Vous pouvez associer un sigle à votre jeu de données. Par exemple, à la [base des entreprises et de leurs établissements](https://www.data.gouv.fr/fr/datasets/5862206588ee38254d3f4e5e), produite par l’INSEE, est associé le sigle SIRENE. Les lettres qui composent votre sigle n’ont pas besoin d’être séparées par des points.
+
+* **Mauvais sigle** : I.R.V.E.
+* **Bon sigle** : IRVE
```
:+1:

bboulesteix replied:
🙂

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+### Licence
+
+Nous vous conseillons de choisir une licence sous laquelle publier vos données, mais rien ne vous y oblige. Si vos données sont ouvertes, nous vous recommandons la licence ouvert ; si elle contiennent une obligation de réciprocité, la licence oDbL convient bien. En choisissant une licence, vous permettez au public de savoir s’il peut adapter et réutiliser vos données et sous quelles conditions.
```
Peut-être ajouter un lien vers https://www.data.gouv.fr/licenses pour donner un peu de contexte

bboulesteix replied:
J’ai prévu d’ajouter un article d’aide au sujet des licences (voir #36), mais tant que cet article n’est pas en ligne, nous pouvons rediriger le lecteur vers : https://www.data.gouv.fr/fr/licences. Tu as raison, ça me semble être une bonne idée. 🙂

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+Nous vous conseillons de choisir une licence sous laquelle publier vos données, mais rien ne vous y oblige. Si vos données sont ouvertes, nous vous recommandons la licence ouvert ; si elle contiennent une obligation de réciprocité, la licence oDbL convient bien. En choisissant une licence, vous permettez au public de savoir s’il peut adapter et réutiliser vos données et sous quelles conditions.
+
+La présence d’une licence est facultative.
```
Mais vivement conseillé.
Il faudrait d'ailleurs peut-être expliquer ce que signifie une absence de licence :thinking:

bboulesteix replied:
Bonne remarque. Que signifie une absence de licence d’ailleurs ? Je ne suis vraiment pas un expert en la matière, je préfère donc poser la question avant d’écrire avec approximation.

romtal replied:
l'absence de licence renvoie directement vers le cadre légal à savoir le code du CRPA...la licence a pour unique objectif d'expliciter le cadre légal pour les quidams :)

bboulesteix replied:
Merci pour cette précision @romtal. Sais-tu ce qu’il est permis de faire avec des données publiées sur data.gouv.fr sans licence ? J’ai peur d’être limité sur ce point. 🤔 

Je trouve bien [quelques articles consacré à la réutilisation des documents administratifs ](https://www.legifrance.gouv.fr/affichCode.do;jsessionid=E4583EBE2857A0DF9056D5D0082F8F2D.tplgfr31s_2?idSectionTA=LEGISCTA000032255212&cidTexte=LEGITEXT000031366350&dateTexte=20181016)dans la loi CADA, mais je ne sais pas si ce sont ces articles qui définissent le cadre légal dont tu parles Romain ? 

Si c’est le cas, alors je peux modifier le texte pour expliquer ce qui se passe si une administration met en ligne des données sans préciser de licence.

bboulesteix replied:
> Si c’est le cas, alors je peux modifier le texte pour expliquer ce qui se passe si une administration met en ligne des données sans préciser de licence.

C’est fait. ✅

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+### Fréquence de mise à jour
+
+Vous devez définir une fréquence _théorique_ de mise à jour pour votre jeu de données. Il s’agit de la fréquence à laquelle vous prévoyez de vous reconnecter sur data.gouv.fr pour mettre à vos données. Cette fréquence est indicative.
```
Non, malheureusement, il s'agit théoriquement de la fréquence de mise à jour des données, la fréquence d'échantillonage ou de collecte et non de la fréquence de publication. Pour l'instant il n'y a qu'un seul champ, il en faudrait un 2e :/

bboulesteix replied:
Je ne suis pas certain de bien saisir la nuance entre la fréquence de mise à jour des données et la fréquence de publication des données. 🤔

Faisons le chemin dans l’autre sens : 

1. Qu’appelons-nous `fréquence de mise à jour` ?
2. Quelle information cherchons-nous à obtenir par l’intermédiaire de ce champ et comment l’utilisons-nous ensuite ?

noirbizarre replied:
Je ne peux pas répondre à 1 et 2 puisque c'est exactement le problème soulevé.

Lorsque tu publies des données, tu ne le fais pas en général à la fréquence à laquelle tu les collectes.
Imaginons un relevé de capteurs quotidien mais dont le producteur ne publie ces données que tous les mois:
- les données sont collectées tous les jours, c'est la fréquence d'échantillonage
- les données sont publiées tous les mois, c'est la fréquence de publication

Initialement, le wording repris est celui de CKAN et correspond à la fréquence d'échantillonage (`accrualPeriodicity` dans DCAT). Malheureusement, le terme étant trop ambiguë (qu'est-ce qui se cache derrière mise à jour), suivant les cas on a l'un ou l'autre dans ce champs.
Et malheureusement, suivant les cas, l'utilisateur s'attends à l'un des deux concepts (en tant que consommateur de données, tu peux vouloir connaitre la granularité de tes données ou bien la fréquence à laquelle tu dois venir chercher les nouvelles publications)

En pratique il faut bien 2 champs, on en a qu'un seul

bboulesteix replied:
Merci pour tes explications. Comme je ne crois pas pouvoir grand chose _au niveau de la documentation_ pour corriger le problème en question, je vais faire une _issue_ sur ce point alors.

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+Vous devez définir une fréquence _théorique_ de mise à jour pour votre jeu de données. Il s’agit de la fréquence à laquelle vous prévoyez de vous reconnecter sur data.gouv.fr pour mettre à vos données. Cette fréquence est indicative.
+
+La fréquence de mise à jour est obligatoire. La date de dernière mise à jour des données est facultative.
```
Ce qui rend ce champ malheureusement incompréhensible

bboulesteix replied:
Voir les questions posées dans la[ discussion précédente](https://github.com/etalab/doc.data.gouv.fr/pull/43#discussion_r225561125).

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+La fréquence de mise à jour est obligatoire. La date de dernière mise à jour des données est facultative.
+
+### Mots-clefs
```
Peut-être enlevé le tiret ici pour avoir la même orthographe que dans le paragraphe (ou l'inverse, rajouter le tiret dans le paragraphe pour avoir la même orthographe que sur le site)

bboulesteix replied:
Tu as raison, pas besoin d’un tiret ici.

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+### Mots-clefs
+
+Vous pouvez ajouter des mots clefs à votre jeu de données. Les mots clefs peuvent indiquer le format de vos données, votre secteur d’activité, votre type de structure, ou encore votre sujet. Les mots clefs apparaissent sur la page de présentation de votre jeu de données. Vous pouvez cliquer sur un mots clef pour voir la liste des autres jeux de données auquel le mot clef en question a été assigné ...
```
Enlever le format de données qui a son propre champ au niveau des ressources. Il servent avant tout à décrire le sujet/métier.

bboulesteix replied:
D’accord, ça me va. 👍

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+### Couverture spatiale
+
+La couverture spatiale de vos données correspond aux zones géographiques couvertes par vos données. Si vos données concerne une ville ou un pays en particulier, c’est ici que vous précisez le nom de la ville ou celui du pays en question. Si vos données concernent plusieurs zones géographiques, vous pouvez les ajouter les unes à la suite des autres, comme s’il s’agissait de mots clefs.
```
"Si ~vos~ ces" pour éviter la répétition

bboulesteix replied:
C’est corrigé, merci. ✅

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+* **fichier principal** : désigne tous les fichiers qui contiennent vos données brutes ;
+* **documentation** : désigne les fichiers annexes, qui contiennent généralement des explications au sujet de vos fichiers principaux ;
+* **mise à jour** : permet d’indiquer que le fichier importé existe déjà sur data.gouv.fr, mais dans une version antérieure, datée ;
```
Cela correspond plutôt à une mise à jour différentielle, un patch à appliquer sur un **fichier principal**

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+Là encore, deux cas à distinguer :
+
+* si vous importez votre fichier, vous n’avez rien à faire, car nous calculons la taille de votre ressource pour vous ;
+* si vous référencez un fichier hébergé ailleurs, indiquez ici sa taille (en bytes).
```
Octets en français

bboulesteix replied:
Bon point. C’est corrigé. ✅

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+Le format du fichier que vous ajoutez. Deux possibilités :
+
+* si vous importez votre fichier, nous détectons automatiquement son format. Vous n’avez donc rien à faire ;
+* si vous référencez un fichier externe, précisez ici son format, c’est-à-dire l’extension du nom de fichier.
```
Pas forcément, le shapefile a une extension en `.zip`, le geojson est souvent en `.json`

bboulesteix replied:
D’accord, je ne savais pas. C’est corrigé dans le texte, merci. ✅

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+Deux cas doivent ici être distingués :
+
+* si vous importez votre fichier, vous n’avez rien à faire, car la somme de contrôle est créée par data.gouv.fr ;
+* si vous référencez un fichier externe, vous pouvez indiquer ici votre propre somme de contrôle. Le menu déroulant permet de sélectionner l’algorithme utilisé pour créer la somme de contrôle, parmi : `sha1`, `sha2`, `sha256`, `md5`, `crc`.
```
recommandé d'utiliser `sha256` s'il y a plusieurs choix ou à la limite `sha1`, `md5` et `crc` sont là pour au cas où mais pas conseillés quand il y a le choix.

bboulesteix replied:
D’accord, j’ai ajouté notre recommandation dans le texte. Merci. ✅

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+
+### Somme de contrôle
+
+La somme de contrôle permet de calculer l’empreinte d’un fichier, pour s’assurer que ce dernier ne varie pas au gré des copies, transferts, ou restaurations.
```
~pour s’assurer que ce dernier ne varie pas~ pour vérifier son intégrité ?

bboulesteix replied:
Que signifie _l’intégrité_ d’un fichier ? Ce terme me paraît hélas relativement technique, j’ai donc cherché à décrire l’utilité d’une somme de contrôle au moyen d’une périphrase, certes plus longue, mais moins pointue à mon avis.

noirbizarre replied:
Je comprends bien l'intention, mais le fait de "varier" sous-entend que le fichier est quand même valide. L'intégrité, par oppposition à corruption, s'attache au fichier dans sa globalité et pas au données structurées qu'il contient.
L'intégrité est un terme bien précis et bien défini lorsqu'il s'agit de fichier ou de données (n'a pas la même signification dans les deux contextes). Mais effectivement il mérite d'être expliqué (plutôt que masqué).
Peut-être que https://fr.wikibooks.org/wiki/Partage_de_fichiers_sur_Internet/V%C3%A9rifier_l%27int%C3%A9grit%C3%A9_d%27un_fichier peut servir de base.

Idée en passant:est-ce qu'on a pas simplement besoin d'un glossaire ? Comme ça, hop, on défini intégrité dedans et un simple lien fait l'affaire

noirbizarre commented on articles/_jeux-de-donnees/publier-un-jeu-de-donnes.md:
```diff
@@ -0,0 +1,220 @@
+Une fois sur la page du jeu de données à modifier :
+
+1. Naviguez jusqu’au bloc intitulé **Ressources** ;
+2. Cliquez sur le bouton **Ajouter** ;
```
Il est aussi possible de faire un drag'n'drop du fichier sur le bloc

bboulesteix replied:
C’est vrai, tu as raison. Je ne parle pas du _glisser/deposer_ ici, car le clic sur le bouton **Ajouter** est un équivalent plus accessible et plus largement utilisable (dans les très vieux navigateurs notamment). Cela dit, j’aime bien l’idée de proposer des raccourcis pour les navigateurs plus modernes ou les utilisateurs plus expérimentés. 🙂

noirbizarre replied:
> dans les très vieux navigateurs notamment

Aucun navigateur sans le support du drad-n-drop ne pourra afficher correctement l'interface d'admin de toute façon. On ne supporte officiellement que les navigateurs encore officiellement supportés par leurs éditeurs, et ils ont tous les support du drag'n'drop.

> les utilisateurs plus expérimentés

Ça n'aide pas forcement que les plus expérimentés. Le cas classique c'est celui qui doit importer X fichiers à la main, il sera content de savoir qu'il peut le faire en une seule fois par drag'n'drop plutôt que de faire X fois **Ajouter**, être bloqué par l'upload dans la modale, attendre, valider et recommencer.

C'est vraiment pour une question pratique. Après ça peut-être rajouté après sous forme de note: "vous pouvez aussi utiliser le glissé-déposé sur le bloc ressource pour faciliter l'envoie de plusieurs ressources", enfin qqch dans ce genre

bboulesteix replied:
> Le cas classique c'est celui qui doit importer X fichiers à la main, il sera content de savoir qu'il peut le faire en une seule fois par drag'n'drop plutôt que de faire X fois Ajouter, être bloqué par l'upload dans la modale, attendre, valider et recommencer.

Tu as raison sur ce point, mais cet argument doit à mon avis être nuancé. Je m’explique : l’intérêt du glissé/déposé est assez limité ici, car la modale d’ajout d’une ressource ne permet pas d’éditer les informations de **plusieurs** fichiers **à la fois**, à la volée. Seul le premier fichier importé peut-être modifié dans la modale.

![screenshot](https://user-images.githubusercontent.com/3286488/47345474-1e3f0b80-d6ab-11e8-9393-7b674ade99d9.png)

Dit autrement, en admettant que je fasse un glissé/déposé pour ajouter deux ressources à un jeu de données, je ne vais pouvoir éditer que les informations qui concernent le _premier_ fichier importé. Si je veux _aussi_ modifier les informations du _deuxième_ fichier, je dois refaire un tour par la fiche du _dataset_, pour cliquer sur le titre de mon second fichier dans le but d’ouvrir la modale qui va me permettre d’en éditer les informations…

Cela dit, le problème souligné ci-dessus peut être contourné en faisant **un** glisser/déposer pour **chaque** fichier, un par un, au coup par coup. Mais dans ce cas, l’opération est à peine plus rapide qu’une série de clics sur le bouton **Ajouter**.

noirbizarre commented on articles/_jeux-de-donnees/supprimer-un-jeu-de-donnees.md:
```diff
@@ -0,0 +1,67 @@
+---
+# Supprimer un jeu de données ou une ressource
+
+Vous pouvez supprimer un jeu de données, ou l’une des ressources qui le compose, si vous êtes l’auteur du jeu de données en question, ou si vous appartenez à l’organisation qui en est à l’origine. Supprimer un jeu de données ou une ressource est irréversible. Un jeu de données ou une ressource supprimé peut rester en ligne jusqu’à 24 heures supplémentaires avant de disparaître complètement.
```
Peut-être inverser les 2 dernières phrases vu que c'est la purge du jeu de données qui est irréversible. Jusqu'à ce que ce soit le cas, il peut maintenant le restaurer avec le bouton dédié.
Peut-être remplacer "jusqu'à 24 heures supplémentaire" par jusqu'à la fin de la journée (ou plutôt il sera définitivement supprimé d'ici la fin de la journée).
Préciser que durant ce laps de temps il ne sera visible que par ses propriétaires et les administrateurs de la plateforme

bboulesteix replied:
Merci pour ces précisions. 👍 Je ne suis pas certain qu’il soit si utile que ça d’expliquer que la suppression d’un jeu de données est l’antichambre d’une purge quotidienne — c’est de la cuisine interne et ça réduit la portée de la suppression en elle-même à mes yeux.

Cela dit, je me réjouis d’apprendre qu’une suppression intempestive est « rattrapable » à condition de réagir vite, ça pourra servir si un producteur nous écrit pour récupérer un jeu de données supprimé. 🙂 En somme, utilisons cette marge de manœuvre comme un garde-fou, pas comme un moyen d’inciter les producteurs à supprimer des jeux de données à la légère.

noirbizarre commented on articles/_jeux-de-donnees/supprimer-un-jeu-de-donnees.md:
```diff
@@ -0,0 +1,67 @@
+
+## Supprimer une ressource
+
+La suppression d’une ressources n’entraîne pas la suppression du jeu de données dont la ressource fait partie.
```
Préciser que dans le cas d'un fichier uploadé sur la plateforme, il reste disponible (pour ceux qui en dépendent) mais qu'il ne sera plus visible sur la fiche

bboulesteix replied:
Merci pour cette précision.

**Questions** : 

1. Qu’est-ce que cela signifie qu’un fichier _reste disponible pour ceux qui en dépendent_ ? 
2. À partir de quand un utilisateur est-il considéré comme dépendant vis-à-vis d’un fichier ?

Je ne suis pas certain de bien comprendre ce cas particulier.

noirbizarre replied:
1. Le fichier est toujours accessible à l'URL qu'il avait. Ça permet à quiconque ayant automatisé quoique ce soit sur cette URL de ne pas avoir de casse. C'est encore plus vrai avec les producteurs qui changent de format de données tous les quatre matins.
2. À partir du moment où il intègre le téléchargement de ce fichier dans n'importe quelle tâche automatisée ou à minima scriptée. C'est aussi le cas des dataviz qui incluent la récupération du fichier au démarrage. Ça évite par exemple d'avoir un graph appelé "resultat 2000-2017" alors que le producteur à mis à jour la ressource avec ceux de 2018. Cas plus banal, ça évite les 404 pour ceux qui ont simplement cité un fichier dans un article.

bboulesteix replied:
D’accord, merci pour ces précisions. Si j’ai bien suivi, la suppression d’un fichier n’en est pas vraiment une, en tout cas pas au sens où on entend le mot « suppression » quand on parle de _suppression d‘un tweet_ par exemple. Il s’agit plutôt, dans notre cas, d’une désindexation, d’un déréférencement.

noirbizarre commented on articles/_jeux-de-donnees/telecharger-une-ressource.md:
```diff
@@ -0,0 +1,15 @@
+2. Repérez la ressource à obtenir et cliquez sur le bouton **Télécharger** qui s’y rapporte ;
+3. Votre téléchargement commence.
+
+S’il la ressource qui vous intéresse est un fichier CSV, vous pouvez prévisualiser le CSV avant de le télécharger en cliquant sur le bouton **Prévisualiser**.
```
Ça n'est pas que pour les CSV, aujourd'hui c'est majoritairement les fichiers geo et en provenance d'OpenDataSoft qui ont des previews.
Peut-être juste signaler que lorsque c'est supporté le bouton prévisualiser est présent.

noirbizarre commented on articles/_jeux-de-donnees/telecharger-une-ressource.md:
```diff
@@ -0,0 +1,15 @@
+---
+# Télécharger une ressource
+
+Quand vous consultez un jeu de données sur data.gouv.fr, les ressources qu’ils contient sont hébergées sur un serveur. Pour ajouter les ressources sur votre propre machine, vous devez les télécharger. Il n’est pas nécessaire de posséder un compte sur data.gouv.fr pour télécharger des données qui y sont répertoriées.
```
Pour ~ajouter~ récupérer les ressources...

bboulesteix replied:
Je viens de remplacer le verbe qui figurait dans le texte par celui que tu suggères, merci. ✅

noirbizarre commented on articles/_jeux-de-donnees/telecharger-une-ressource.md:
```diff
@@ -0,0 +1,15 @@
+---
+# Télécharger une ressource
+
+Quand vous consultez un jeu de données sur data.gouv.fr, les ressources qu’ils contient sont hébergées sur un serveur. Pour ajouter les ressources sur votre propre machine, vous devez les télécharger. Il n’est pas nécessaire de posséder un compte sur data.gouv.fr pour télécharger des données qui y sont répertoriées.
```
C'est peut-être l'endroit pour préciser la différence entre le lien direct qui ne bouge pas lors d'une mise à jour ou suppression de ressource et le permalien qui pointe toujours vers la dernière version et disparait à la suppression de la ressource

bboulesteix replied:
Pourquoi pas. Même si je pensais plutôt définir ces termes dans le _glossaire_ (voir #36). Avant de me lancer dans la rédaction de ces définitions, j’aimerais que nous nous entendions sur les termes employés ici.

Peux-tu me confirmer que :

1. Ce que tu appelles _le lien direct qui ne bouge pas_ correspond en fait à ce qui s’appelle **URL** dans l’interface, quand on clique sur une ressource (voir la capture ci-dessous) ?
2. Ce que tu appelles _permalien qui pointe toujours vers la dernière version et disparait à la suppression de la ressource_ correspond en fait à ce qui s’appelle **Dernière URL** dans l’interface, quand on clique sur une ressource (voir la capture ci-dessous)  ?

Enfin, je serais curieux de connaître les cas dans lesquelles la **Dernière URL** est préférable à une **URL** pure et simple. 🤔 Je me demande si beaucoup d’utilisateurs saisissent la nuance et en tirent profit.

<img width="592" alt="capture d ecran 2018-10-16 17 03 18" src="https://user-images.githubusercontent.com/3286488/47026284-63bc7f80-d165-11e8-9000-7aa8041c52e1.png">

noirbizarre commented on articles/_organisations/creer-une-organisation.md:
```diff
@@ -0,0 +1,80 @@
+---
+# Créer une organisation
+
+Une organisation est une entité au travers de laquelle plusieurs utilisateurs peuvent collaborer sur des jeux de données. Une organisation peut contenir plusieurs utilisateurs ; un même utilisateur peut quant à lui appartenir à plusieurs organisations. Les administrateurs et les éditeurs d’une organisation peuvent en contrôler les membres et les publications au moyen d’une section spécifique de l ...
```
Seuls les administrateurs peuvent contrôler les membres

bboulesteix replied:
Je te crois, mais dans ce cas, il doit y avoir un pépin, car je ne suis qu’un simple _éditeur_ au sein de l’organisation Etalab et j’ai pourtant la possibilité de :

* ajouter des membres dans l’organisation ;
* approuver les demandes d’adhésion en attente ;
* révoquer des membres de l’organisation (voir la capture ci-dessous).

Le contrôle des membres est-il donc vraiment réservé aux administrateurs d’un organisation ? 🤔 

<img width="298" alt="capture d ecran 2018-10-16 15 49 21" src="https://user-images.githubusercontent.com/3286488/47021167-366ad400-d15b-11e8-826a-f3f432334402.png">

noirbizarre replied:
Mais tu es administrateur sur la plateforme, tu as tous les droits indépendamment de ton rôle dans les différentes orga (d'où l’intérêt d'avoir 2 comptes pour tester et documenter).
Il y a aussi un [bug sur les notifs de transfert à corriger sur ce sujet](https://github.com/opendatateam/udata/issues/794) avant que tu ne pose la question :wink:

bboulesteix replied:
Ahah, en effet. 🙂 Je ne savais pas qu’il y avait une notion de **super administrateur**, mais je comprends mieux désormais. J’ai bien un second compte pour faire mes essais, mais il faut que je m’en serve davantage, tu as raison. Merci pour tes lumières en tout cas.

noirbizarre commented on articles/_organisations/creer-une-organisation.md:
```diff
@@ -0,0 +1,80 @@
+
+Une organisation est une entité au travers de laquelle plusieurs utilisateurs peuvent collaborer sur des jeux de données. Une organisation peut contenir plusieurs utilisateurs ; un même utilisateur peut quant à lui appartenir à plusieurs organisations. Les administrateurs et les éditeurs d’une organisation peuvent en contrôler les membres et les publications au moyen d’une section spécifique de l ...
+
+La création d’une organisation est gratuite. Elle nécessite la création d’un compte utilisateur, car une organisation doit avoir au moins un membre.
```
Il n'y a pas de contrainte sur le nombre de membres. Lorsque le dernier membre quitte l'organisation ou supprime son compte, l'organisation existe toujours

bboulesteix replied:
Merci pour ces précisions.

> Il n'y a pas de contrainte sur le nombre de membres

Je vais rajouter cet élément dans l’article en question.

> Lorsque le dernier membre quitte l'organisation ou supprime son compte, l'organisation existe toujours.

Excellent point ! Merci de m’y faire penser. Cette information figurera dans l’article consacré à la suppression d’une organisation, article que je n’ai pas encore rédigé, mais qui est bien prévu (voir #36). 🙂

noirbizarre commented on articles/_organisations/creer-une-organisation.md:
```diff
@@ -0,0 +1,80 @@
+Nous vous conseillons de créer une organisation si vous souhaitez :
+
+* publier des jeux de données pour le compte d’une administration, une collectivité, ou une entité indépendante ;
+* permettre à des utilisateurs différents de publier des jeux de données sous le même nom, la même bannière.
```
ajouter que seule une organisation peut être certifiée

bboulesteix replied:
Ah oui, tu fais bien de me le rappeler, merci. 👍

noirbizarre commented on articles/_organisations/creer-une-organisation.md:
```diff
@@ -0,0 +1,80 @@
+Les formats d’image suivant sont acceptés :
+* png ;
+* jpg ;
+* jpeg.
```
Normalement `jpeg` est le format, `jpg` son extension. Peut-être juste garder l'un des deux, même si le serveur est capable de détecter cette erreur

bboulesteix replied:
Merci pour cette précision. Tu as raison sur ce coup. Si je mentionne séparément `jpg` et `jpeg`, c’est que j’estime que le lien entre ces deux extension n’est pas acquis pour tout le monde — il l’est à l’évidence pour toi. 🙂 Dit autrement, je voudrais éviter que quelqu’un qui possède un fichier `fichier.jpg` sur sa machine renonce à l’importer sur data.gouv.fr sous prétexte que nous ne parlons que des fichiers `jpeg` dans notre article d’aide. Sans compter que certaines applications de photo créent des fichiers `jpg` au moment de l’exportation, là où d’autres produisent des `jpeg`… En gros, je vise large, je sais, mais c‘est délibéré.

noirbizarre replied:
Effectivement. Dans ce cas là peut-être mettre un seul point avec `jpg/jpeg` ?

bboulesteix replied:
C’est mieux, tu as raison. 👍 Corrigé. ✅

noirbizarre commented on articles/_reutilisations-et-discussions/associer-une-reutilisation-a-un-jeu-de-donnees.md:
```diff
@@ -0,0 +1,87 @@
+
+* ce que la réutilisation cherche à montrer ;
+* les conclusions que la réutilisation permet de tirer ;
+* l’auteur de la réutilisation.
```
Ajouter la méthode de création, les algos... comment ils l'ont faite et peut-être quel usage des données il est fait.
La réutilisation n'est pas toujours un article, par exemple, l'objet connecté ne cherche pas à montrer à quelque chose et ne tire pas de conclusion.

noirbizarre commented on articles/_reutilisations-et-discussions/associer-une-reutilisation-a-un-jeu-de-donnees.md:
Préciser qu'il est possible de rajouter des jeux de données à une réutilisation existante depuis la page d'un jeu de données en cliquant sur "ajouter une réutilisation"

bboulesteix replied:
Hum, de mon côté, si je clique sur **Ajouter une réutilisation** depuis la page d’un jeu de données, je me retrouve dans le tunnel de création d’une _nouvelle_ réutilisation. Je n’ai pas la possibilité de choisir une réutilisation _existante_ à laquelle ajouter le jeu de données en question. Mais j’ai peut-être raté un épisode. 🤔

noirbizarre replied:
Il faut que tu ai une réutilisation existante sinon effectivement il te redirige vers le parcours de création :wink:

bboulesteix replied:
Je dois mal comprendre, car je ne vois pas où tu veux en venir.

Exemple :

1. Je prends une réutilisation existante, créée par une organisation dont je suis administrateur : https://www.data.gouv.fr/fr/reuses/evolution-des-disciplines-des-jeux-olympiques/
2. Je prends un jeu de données créé par Etalab, auquel est déjà associé une autre réutilisation : https://www.data.gouv.fr/fr/datasets/statistiques-dusage-de-lapi-de-geocodage-adresse-data-gouv-fr/
3. Je clique sur **Ajouter une réutilisation** sur la page du jeu de données en question ;
4. Je me retrouve inexorablement dans le parcours de création.

<img width="1440" alt="capture d ecran 2018-10-23 10 42 16" src="https://user-images.githubusercontent.com/3286488/47347834-5eed5380-d6b0-11e8-9931-312bdc599b10.png">

**Question @noirbizarre**
Peux-tu m’indiquer les étapes qui permettent de reproduire le cas que tu indiques, à savoir ajouter un jeu de données _lambda_ à une réutilisation _lambda_ depuis la page du jeu de données ? Je pose la question car, pour moi, ce que tu dis ne semble possible que depuis l’interface administrateur.

<img width="597" alt="capture d ecran 2018-10-23 10 44 49" src="https://user-images.githubusercontent.com/3286488/47347977-bdb2cd00-d6b0-11e8-922f-c69365ce3eba.png">

noirbizarre replied:
Tu n'as pas cette modale qui s'affiche ?
![screenshot-www data gouv fr-2018 10 23-17-03-44](https://user-images.githubusercontent.com/15725/47370529-38e2a600-d6e6-11e8-8596-973fbde9065c.png)

bboulesteix replied:
Non, je n’ai jamais vu cette modale @noirbizarre. Mais je m’y suis peut-être mal pris. Quelles étapes as-tu suivi pour faire apparaître cette modale ?

noirbizarre replied:
J'ai juste cliqué sur ajouter un réutilisation depuis la page du jeu de données, sous les discussions.
Mais bon, on va merger sans ça et identifier pourquoi elle ne s'affiche pas chez toi.

bboulesteix commented:
Merci d’avoir pris le temps de me relire @noirbizarre, c’est vraiment très gentil de ta part. 👏  Je vais répondre à tes commentaires dans la journée.

Commit:
Modifie publier-un-jeu-de-donnes.md

Ajoute de l’emphase sur la pondération du titre dans les facteurs pris en compte lors de l’indexation des jeux de données.

Commit:
Modifie publier-un-jeu-de-donnes.md

Ce _commit_ ajoute un lien vers https://www.data.gouv.fr/fr/licences dans le paragraphe consacré au choix d’une licence.

Commit:
Modifie publier-un-jeu-de-donnes.md

Supprime le tiret de `mots-clefs`.

Commit:
Modifie publier-un-jeu-de-donnes.md

Supprime la référence au format des données dans les conseils donnés pour bien choisir des mots clefs.

Commit:
Modifie creer-une-organisation.md

Ajoute une référence au nombre maximal de membres que peut accueillir une organisation, nombre qui n’est pas limité.

Commit:
Modifie differences-utilisateur-organisatoion.md

Pour ajouter une référence à la certification, qui est accessible aux organisations, mais pas aux utilisateurs individuels.

Commit:
Modifie associer-une-reutilisation-a-un-jeu-de-donnees.md

Ce _commit_ ajoute des exemples de choses à inclure dans la description d’une réutilisation.

Commit:
Modifie publier-un-jeu-de-donnes.md

Ce commit corrige une coquille et supprime une répétition.

Commit:
Modifie publier-un-jeu-de-donnes.md

Ce commit remplace un mot anglais (bytes) par un mot français équivalent (octets).

Commit:
Modifie publier-un-jeu-de-donnes.md

Ce commit corrige une approximation au sujet des formats de données.

Commit:
Modifie publier-un-jeu-de-donnes.md

Ce commit ajoute une recommandation sur l’algorithme à utiliser pour la création d’une somme de contrôle.

Commit:
Modifie supprimer-un-jeu-de-donnees.md

Ajoute des précisions sur les conséquences de la suppression d’un jeu de données.

Commit:
Modifie publier-un-jeu-de-donnes.md

Ce commit corrige une approximation au sujet de la mise à jour des données.

Commit:
Modifie telecharger-une-ressource.md

Ce commit ajoute des précisions au sujet de la prévisualisation des fichiers.

Commit:
Modifie telecharger-une-ressource.md

Ce commit change un verbe.

bboulesteix commented:
Encore merci pour tes nombreuses suggestions @noirbizarre. Elles m’aident beaucoup, vraiment. 👏 

De mon côté, j’ai :

* corrigé les textes sur lesquels portaient tes remarques lorsque des changements s’imposaient, ce qui prend la forme de nouveaux _commits_ sur ma branche ;
* ajouté des questions en réponse à tes commentaires, dans les discussions déjà ouvertes, lorsque j’avais besoin de précisions de ta part avant de pouvoir modifier les fichiers dont tu parlais.

Toutes les discussions encore ouvertes attendent donc une réponse de ta part. 🙏

noirbizarre reviewed (commented):
Par contre on ne peut pas mettre de commentaire sur les binaires, mais il faudrait enlever les `.DS_Store` de MacOS de la PR (et les rajouter au `.gitignore`)

noirbizarre commented on articles/_gestion-du-compte/fermer-un-compte.md:
```diff
@@ -0,0 +1,15 @@
+title: Fermer un compte
+slug: fermer-un-compte
+---
+# Fermer un compte
```
**Supprimer mon compte** à la place de **Fermer un compte**. Cette subtilité est importante:
- c'est le vocabulaire que les utilisateurs utilisent dans les mails qu'ils nous envoient (donc ce qu'ils vont chercher)
- pour être conformité avec la CNIL, le compte doit être supprimé à la demande l'usager, par opposition à la simple fermeture qui implicitement ne comprends pas la suppression et peut laisser croire qu'il peut-être rouvert.

noirbizarre commented on articles/_gestion-du-compte/voir-ou-modifier-un-compte-existant.md:
```diff
@@ -0,0 +1,45 @@
+title: Voir ou modifier un compte
+slug: voir-ou-modifier-un-compte-existant
+---
+# Voir ou modifier un compte
```
**votre** plutôt que **un** puisqu'on ne peut modifier que sont propre compte (hors administrateurs biensur)

bboulesteix replied:
Bon point, c’est corrigé. ✅

noirbizarre commented on articles/_jeux-de-donnees/difference-jeu-de-donnees-et-ressource.md:
```diff
@@ -0,0 +1,38 @@
+title: Différences entre un jeu de données et une ressource
+slug: difference-jeu-de-donnees-et-ressource
+---
+# Différences entre un jeu de données et une ressource
```
Qu'est-ce qu'un jeu de données et une ressource ?
Le titre actuel implique des similitudes entre les deux et une comparaison ce qui n'est pas le cas.

bboulesteix replied:
Certes, pourquoi pas, mais enfin il s’agit dans les deux cas de « contenus » publiés sur `data.gouv.fr`, le titre ne me semble donc pas si trompeur que ça. 🙂 

Par ailleurs, je préfère [éviter les titres d’articles à la forme interrogative](https://gds.blog.gov.uk/2013/07/25/faqs-why-we-dont-have-them/), car ils ont tendance à décaler les mots clefs vers la fin de la phrase (ce qui en ralentit la compréhension).

Commit:
Modifie et renomme supprimer-un-compte.md

Ajoute un paragraphe sur ce que deviennent les contenus associés à un compte supprimé.

Commit:
Modifie publier-un-jeu-de-donnes.md

Ajoute une ligne sur l’importance de publier au nom d’une organisation

Commit:
Renomme voir-ou-modifier-votre-compte.md

Modifie et change le titre de `voir-ou-modifier-votre-compte.md`.

Commit:
Modifie creer-une-organisation.md

Correction typographique.

bboulesteix committed:
Supprime le fichier .DS_Store

Commit:
Modifie publier-un-jeu-de-donnes.md

Ajoute un lien vers Legifrance pour traiter le cas où les données seraient publiées sans licence.

Commit:
Modifie supprimer-un-jeu-de-donnees.md

Ajoute un paragraphe sur le devenir des ressources supprimées.

bboulesteix commented:
Encore merci d’avoir pris le temps de me répondre à nouveau @noirbizarre. Il ne reste plus qu’une [seule discussion ouverte](
https://github.com/etalab/doc.data.gouv.fr/pull/43#discussion_r227285721), si je ne dis pas de bêtise, et elle n’empêche pas de _merger_ le reste du travail.

> Par contre on ne peut pas mettre de commentaire sur les binaires, mais il faudrait enlever les .DS_Store de MacOS de la PR (et les rajouter au .gitignore).

C’est fait. ✅ Désolé pour le bruit.

> This branch has conflicts that must be resolved

Je ne comprends pas sur quoi porte le conflit et la raison pour laquelle il persiste. 🤔

bboulesteix committed:
Merge remote-tracking branch 'upstream/master'

bboulesteix committed:
Merge branch 'add-some-help-articles' of https://github.com/bboulesteix/doc.data.gouv.fr into bboulesteix-add-some-help-articles

bboulesteix committed:
Merge branch 'bboulesteix-add-some-help-articles'

bboulesteix committed:
Corrige le conflit

bboulesteix committed:
Corrige un peu mieux le conflit

bboulesteix committed:
Merge branch 'master' into add-some-help-articles
